Samples/excel-add-in-chart-analyze-data/src/taskpane/taskpane.ts

```diff
@@ -0,0 +1,588 @@
+
+      const range4 = sheet.getRange("A38");
+      range4.formulas = [["=B5"]];
+      range4.format.autofitColumns();
+
+      const range11 = sheet.getRange("B37");
+      range11.formulas = [['=SUMIF($B$2:$B$33,"=Park",$C$2:$C$33)']];
+      range11.format.autofitColumns();
+
+      const range12 = sheet.getRange("C37");
+      range12.formulas = [['=SUMIF($B$2:$B$33,"=Park",$D$2:$D$33)']];
+      range12.format.autofitColumns();
+
+      const range21 = sheet.getRange("B38");
+      range21.formulas = [['=SUMIF($B$2:$B$33,"=Beach",$C$2:$C$33)']];
+      range21.format.autofitColumns();
+
+      const range22 = sheet.getRange("C38");
+      range22.formulas = [['=SUMIF($B$2:$B$33,"=Beach",$D$2:$D$33)']];
+      range22.format.autofitColumns();
+
+      const range33 = sheet.getRange("L36");
+      range33.formulas = [["=MAX(H2:H33)"]];
+      range33.format.autofitColumns();
+
+      const range36 = sheet.getRange("L37");
+      range36.formulas = [["=AVERAGE(H2:H33)"]];
+      range36.format.autofitColumns();
+
+      const range34 = sheet.getRange("M36");
+      range34.formulas = [["=INDEX(A2:A33,MATCH(L36,H2:H33,0),0)"]];
+      range34.format.autofitColumns();
+
+      let dateRange = sheet.getRange("A2:A33");
+
+      let formatdate = [];
+      let formatdateitem = ["m/d"];
+      for (let i = 0; i < 32; i++) {
+        formatdate.push(formatdateitem);
+      }
+      dateRange.numberFormat = formatdate;
+
+      let numRange1 = sheet.getRange("C2:C33");
+      let numRange2 = sheet.getRange("D2:D33");
+      let numRange3 = sheet.getRange("H2:H33");
+      let formatnumber = [];
+      let formatnumberitem = ["###,0"];
+      for (let i = 0; i < 32; i++) {
+        formatnumber.push(formatnumberitem);
+      }
+      numRange1.numberFormat = formatnumber;
+      numRange2.numberFormat = formatnumber;
+      numRange3.numberFormat = formatnumber;
+
+      if (Office.context.requirements.isSetSupported("ExcelApi", 1.2)) {
+        sheet.getUsedRange().format.autofitColumns();
+        sheet.getUsedRange().format.autofitRows();
+      }
+      sheet.activate();
+      sheet.gridlines = false;
+
+      sheet.onChanged.add(onDataChanged);
+      await context.sync();
+      showStatus('Success for "Add Sample data"', false);
+    });
+  } catch (error) {
+    showStatus(error, true);
+  }
+}
+
+async function onDataChanged(event) {
+  try {
+    await Excel.run(async (context) => {
+      console.log("data changed");
+      highlightSales();
+
+      await context.sync();
+    });
+  } catch (error) {
+    showStatus(error, true);
+  }
+}
+
+function showStatus(message, isError) {
+  let status = document.getElementById("status");
+  // Clear previous content
+  status.innerHTML = "";
+
+  // Create the container div
+  let statusCard = document.createElement("div");
+  statusCard.className = `status-card ms-depth-4 ${isError ? "error-msg" : "success-msg"}`;
+
+  // Create and append the first paragraph
```

Comment:
This is more of a header than a paragraph, right?
```suggestion
  // Create and append the first paragraph.
```